api/provider/aws/logging.go

```diff
@@ -16,6 +16,73 @@ const (
 	MAX_DESCRIBE_STREAMS_COUNT = 1000
 )
 
+func GetLogsFromCloudTrail(
```

Comment:
Keep things DRY: you should wrap/update/integrate with [GetLogsFromTaskARNs](https://github.com/quintilesims/layer0/blob/api-refactor/api/provider/aws/logging.go#L19) instead of using copy+paste for so much of the implementation.